src/components/Blogs/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react";
+import { Container } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const POSTS_PER_PAGE = 9;
+const BLOG_API_URL = `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts?per_page=${POSTS_PER_PAGE}`;
+
+export default function Blogs() {
+  const { t } = useTranslation();
+  const [blogPosts, setBlogPosts] = useState([]);
+
+  const fetchBlogPosts = async () => {
+    const data = await fetch(BLOG_API_URL);
+    const fetchedBlogPosts = await data.json();
+    setBlogPosts(fetchedBlogPosts);
+  };
+  useEffect(() => {
+    fetchBlogPosts();
+  }, []);
+
+  return (
+    <Container>
+      {blogPosts.map((blogPost) => {
+        return (
+          <div className="blogsSection">
+            <div className="shadowEffect"></div>
+            <h3
+              className="blogPostTitle"
+              dangerouslySetInnerHTML={{
+                __html: t(blogPost.title.rendered),
```

Comment:
hmm I'm confused. this isn't coming from the translations file. work with hiba or jennet here to solve this, I'm not sure what this page is supposed to be so I can't advise

Reply:
You are right this is not from the translations file, this one is being fetched from WP API to show only the titles for the posts. I had to use `dangerouslySetInnerHTML` because of the data was fetched with extra `&nbsp;` that wasn't supposed to be shown.

Reply:
right but it has a t() around it -- i think it's not supposed to